StructureDefinition/England-PractitionerRole.xml

```diff
@@ -0,0 +1,353 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="England-PractitionerRole" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/England-PractitionerRole" />
+  <version value="1.0.0" />
+  <name value="EnglandPractitionerRole" />
+  <title value="England PractitionerRole" />
+  <status value="draft" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="This profile defines the UK constraints and extensions on the UK Core FHIR resource [UK Core PractitionerRole](https://simplifier.net/hl7fhirukcorer4/ukcore-practitionerrole)." />
```

Comment:
we should refer to UK Core as FHIR UK Core as per Dave C

Reply:
Pushed change